examples/computed-field/package.json

```diff
@@ -9,9 +9,9 @@
     "build": "tsc"
   },
   "author": "",
-  "license": "Apache-2.0",
+  "license": "UNLICENSED",
```

Comment:
We change licensing of SDK ?

Reply:
It's not the sdk, it's the plugin exemple. It's not open source